src/main/java/com/airlines/dao/AirplaneDaoImpl.java

```diff
@@ -0,0 +1,189 @@
+        try (Connection connection = dataSource.getConnection()) {
+            return findAirplaneByCodeName(codeName, connection);
+        } catch (SQLException e) {
+            throw new DaoOperationException(String.format("Cannot find Airplane by codeName = %s", codeName), e);
+        }
+    }
+
+    private Airplane findAirplaneByCodeName(String codeName, Connection connection) throws SQLException {
+        PreparedStatement selectByIdStatement = prepareSelectByCodeNameStatement(codeName, connection);
+        ResultSet resultSet = selectByIdStatement.executeQuery();
+        resultSet.next();
+        return parseRow(resultSet);
+    }
+
+    private PreparedStatement prepareSelectByCodeNameStatement(String codeName, Connection connection) {
+        try {
+            PreparedStatement selectByIdStatement = connection.prepareStatement(SELECT_AIRPLANE_BY_ID_SQL);
+            selectByIdStatement.setString(1, codeName);
+            return selectByIdStatement;
+        } catch (SQLException e) {
+            throw new DaoOperationException("Cannot prepare statement to select airplane by codeName", e);
+        }
+    }
+
+    private Airplane parseRow(ResultSet resultSet) throws SQLException {
+        return Airplane.builder()
+                .withId(resultSet.getInt("id"))
+                .withCodeName(resultSet.getString("codeName"))
+                .withModel(Model.valueOf(resultSet.getString("model")))
+                .withManufacture(resultSet.getDate("manufactureDate").toLocalDate())
+                .withCapacity(resultSet.getInt("capacity"))
+                .withFlightRange(resultSet.getInt("flightRange"))
+                .withCrewId(resultSet.getInt("crewId"))
+                .build();
+    }
+
+    @Override
+    public List<Airplane> findAll() {
+        try (Connection connection = dataSource.getConnection()) {
+            Statement statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery(SELECT_ALL_AIRPLANE_SQL);
+            return collectToList(resultSet);
+        } catch (SQLException e) {
+            throw new DaoOperationException(e.getMessage());
+        }
+    }
+
+    private List<Airplane> collectToList(ResultSet resultSet) throws SQLException {
+        List<Airplane> accountList = new ArrayList<>();
+        while (resultSet.next()) {
+            Airplane airplane = parseRow(resultSet);
+            accountList.add(airplane);
+        }
+        return accountList;
+    }
+
+    public boolean deleteAirplane(int id) {
+        boolean rowDeleted;
+        try (Connection connection = dataSource.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(DELETE_AIRPLANE_SQL);
+            statement.setInt(1, id);
+            rowDeleted = statement.executeUpdate() > 0;
+            return rowDeleted;
+        } catch (SQLException e) {
+            throw new DaoOperationException(e.getMessage());
```

Comment:
try to point out custom message format("Can not delete airplane with id = %s", s), e)